### ISSUE TITLE
Quicklisp's Deflate Fails to Decompress Due to Block Length Mismatch

### ISSUE THREAD

**Issue by Shinmera** [drmeister/clasp#156]
Quickloading anything that requires a download results in this error on d13d2308ca2ac24ec49a3d0e58f21efa96f91b6c.
```
> (ql:quickload :softdrink)
To load "softdrink":
  Load 4 ASDF systems:
    cl-base64 lquery trivial-indent trivial-mimes
  Install 2 Quicklisp releases:
    lass softdrink
; Fetching #<URL "http://beta.quicklisp.org/archive/lass/2015-06-08/lass-20150608-git.tgz" @0x2ab4b15e0011>
; 22.35KB
==================================================
22,891 bytes in 0.01 seconds (3193.50KB/sec)

Condition of type: DEFLATE-DECOMPRESSION-ERROR
Error detected during deflate decompression: Block length mismatch for stored block: LEN(31580) vs. NLEN(58508)!
Available restarts:
(use :r1 to invoke restart 1)

1. (ABORT) Give up on "softdrink"
2. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[201] TOP-LEVEL.
 File: #P"/home/linus/.cache/shirakumo/autobuild/clasp-testing/.autobuild/d13d2308ca2ac24ec49a3d0e58f21efa96f91b6c/build/clasp/Contents/Resources/lisp/kernel/lsp/top.lsp" (Position #21029)
>> 
```

**Comment by drmeister** [comment]
I think quicklisp works now

**Comment by Shinmera** [comment]
Fix confirmed on 61359697067e4e0a2bb6dab35539ea1534e675cb
```
> (ql:quickload :softdrink)
To load "softdrink":
  Load 4 ASDF systems:
    cl-base64 lquery trivial-indent trivial-mimes
  Install 2 Quicklisp releases:
    lass softdrink
; Fetching #<URL "http://beta.quicklisp.org/archive/softdrink/2014-11-06/softdrink-20141106-git.tgz" @0x2add74ae75b1>
; 7.62KB
==================================================
7,801 bytes in 0.003 seconds (2539.39KB/sec)
; Loading "softdrink"
[package array-utils].............................
```
